src/famfs_lib.c

```diff
@@ -3209,13 +3209,14 @@ famfs_cp(struct famfs_locked_log *lp,
 			char destpath[PATH_MAX];
 			char src[PATH_MAX];
 
-			if (verbose > 1)
+			if (verbose)
```

Comment:
Why did you drop (verbose > 1)? The intent is that this only gets printed if verbose is > 1. If that branch was not covered, it is easy to cover by adding a "famfs cp -vv" (which I suspect we already have)

Reply:
Ok, I had passed the verbose=1 option from the test and it did not cover the path...  Also I thought passing 1 should enable it as I don't see options to be verbose at different log levels (like 1, 2, 3... n), in code as of now anything greater than 1 is essentially same..

Will change it back...